lib/tasks/custom_seed.rake

```diff
@@ -0,0 +1,13 @@
+namespace :db do
+  namespace :seed do
+    Dir[Rails.root.join("db/custom_seeds/*.rb")].each do |filename|
+      task_name = File.basename(filename, ".rb")
+      desc "Seed #{task_name}, based on the file with the same name in `db/seeds/*.rb`"
+      task task_name.to_sym => :environment do
+        break if Rails.env.production?
```

Comment:
This will raise an exception:

```
LocalJumpError: break from proc-closure
```

I suggest using `abort` here. It's part of the `Kernel` module, so available everywhere. It also sets a non-zero exit code, indicating failure. 

```suggestion
        abort if Rails.env.production?
```